error-prone-contrib/src/main/java/tech/picnic/errorprone/refasterrules/OptionalRules.java

```diff
@@ -373,7 +370,12 @@ Optional<R> after(Optional<T> optional, Function<? super S, Optional<? extends R
   /** Prefer {@link Optional#or(Supplier)} over more verbose alternatives. */
   static final class OptionalOrOtherOptional<T> {
     @BeforeTemplate
-    @SuppressWarnings("NestedOptionals")
+    @SuppressWarnings({
+      "LexicographicalAnnotationAttributeListing" /* `key-*` entry must remain last. */,
+      "NestedOptionals" /* This violation will be rewritten. */,
+      "OptionalOrElse" /* Here `optional2` is a stand-in for expressions that may require computation. */,
+      "key-to-resolve-AnnotationUseStyle-and-TrailingComment-check-conflict"
+    })
```

Comment:
Yes, this is getting messy. Maybe those `key-to-resolve-AnnotationUseStyle-and-TrailingComment-check-conflict` entries should start with `zzz-` or something.